packages/react-components/src/lib/ol-map/map.tsx

```diff
@@ -75,20 +102,15 @@ export const Map: React.FC<MapProps> = (props) => {
     if (showMousePosition !== undefined && showMousePosition) {
       map.addControl(
         new MousePosition({
-          coordinateFormat: (coord): string =>
-            format(
-              coord as Coordinate,
-              '{y}°N {x}°E',
-              COORDINATES_FRACTION_DIFITS
-            ),
-          projection: PROJECTION,
+          coordinateFormat: getCoordinateFormatString(projection),
```

Comment:
think default value should be **projection ?? Proj.WGS84**

Reply:
fixed